### ISSUE TITLE
It doesn't work on my RX 9060 XT 16GB

### ISSUE THREAD

**Issue by Wemite** [patientx-cfz/comfyui-rocm#38]
The entire installation went without errors. Before running install.bat, I installed git, ms build tools, and vcredist. When I run comfyui_user.bat, I get this error:

PS C:\comfyui-rocm> .\comfyui-user.bat
::: [comfyui-rocm] starting with these parameters :::
[ --disable-api-nodes --cache-none --disable-smart-memory --disable-pinned-memory --enable-manager-legacy-ui ]

setup plugin alembic.autogenerate.schemas
setup plugin alembic.autogenerate.tables
setup plugin alembic.autogenerate.types
setup plugin alembic.autogenerate.constraints
setup plugin alembic.autogenerate.defaults
setup plugin alembic.autogenerate.comments
[START] Security scan
[DONE] Security scan
## ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2026-05-01 01:04:19.988
** Platform: Windows
** Python version: 3.12.9 (tags/v3.12.9:fdb8142, Feb  4 2025, 15:27:58) [MSC v.1942 64 bit (AMD64)]
** Python executable: C:\comfyui-rocm\python_env\python.exe
** ComfyUI Path: C:\comfyui-rocm
** ComfyUI Base Folder Path: C:\comfyui-rocm
** User directory: C:\comfyui-rocm\user
** ComfyUI-Manager config path: C:\comfyui-rocm\user\__manager\config.ini
** Log path: C:\comfyui-rocm\user\comfyui.log

Prestartup times for custom nodes:
  18.4 seconds: C:\comfyui-rocm\custom_nodes\ComfyUI-Manager

Found comfy_kitchen backend eager: {'available': True, 'disabled': False, 'unavailable_reason': None, 'capabilities': ['apply_rope', 'apply_rope1', 'dequantize_mxfp8', 'dequantize_nvfp4', 'dequantize_per_tensor_fp8', 'quantize_mxfp8', 'quantize_nvfp4', 'quantize_per_tensor_fp8', 'scaled_mm_mxfp8', 'scaled_mm_nvfp4']}
Found comfy_kitchen backend cuda: {'available': True, 'disabled': True, 'unavailable_reason': None, 'capabilities': ['apply_rope', 'apply_rope1', 'dequantize_nvfp4', 'dequantize_per_tensor_fp8', 'quantize_mxfp8', 'quantize_nvfp4', 'quantize_per_tensor_fp8']}
Found comfy_kitchen backend triton: {'available': True, 'disabled': True, 'unavailable_reason': None, 'capabilities': ['apply_rope', 'apply_rope1', 'dequantize_nvfp4', 'dequantize_per_tensor_fp8', 'quantize_mxfp8', 'quantize_nvfp4', 'quantize_per_tensor_fp8']}
Checkpoint files will always be loaded safely.
Unhandled exception caught in c10/util/AbortHandler.h
00007FFA79B2A9BC00007FFA79B19EB0 torch_python.dll!initModule [<unknown file> @ <unknown line number>]
00007FFB6AF719F700007FFB6AF719E0 ucrtbase.dll!terminate [<unknown file> @ <unknown line number>]
00007FFB555F113400007FFB555F10F0 VCRUNTIME140.dll!_FrameUnwindFilter [<unknown file> @ <unknown line number>]
00007FFB5560204C00007FFB55601A90 VCRUNTIME140.dll!memset [<unknown file> @ <unknown line number>]
00007FFB555FE7BF00007FFB555FE720 VCRUNTIME140.dll!_C_specific_handler [<unknown file> @ <unknown line number>]
00007FFB6D50623F00007FFB6D5061A0 ntdll.dll!_chkstk [<unknown file> @ <unknown line number>]
00007FFB6D3B239700007FFB6D3B1E00 ntdll.dll!RtlLocateExtendedFeature [<unknown file> @ <unknown line number>]
00007FFB6D3AA95100007FFB6D3AA730 ntdll.dll!RtlRaiseException [<unknown file> @ <unknown line number>]
00007FFB6A1779DA00007FFB6A177950 KERNELBASE.dll!RaiseException [<unknown file> @ <unknown line number>]
00007FFB555F526700007FFB555F51D0 VCRUNTIME140.dll!CxxThrowException [<unknown file> @ <unknown line number>]
00007FFA8F93602400007FFA8F935EC0 c10_hip.dll!c10::cuda::c10_cuda_check_implementation [<unknown file> @ <unknown line number>]
00007FFA8F93694C00007FFA8F9368F0 c10_hip.dll!c10::cuda::SetDevice [<unknown file> @ <unknown line number>]
00007FFA79B72AE400007FFA79B65100 torch_python.dll!THPStorage_assertNotNull [<unknown file> @ <unknown line number>]
00007FFA7A2B2A1D00007FFA7A2711F0 torch_python.dll!torch::lazy::GetPythonFrames [<unknown file> @ <unknown line number>]
00007FFB555FF73000007FFB555FF4D0 VCRUNTIME140.dll!_CxxFrameHandler3 [<unknown file> @ <unknown line number>]
00007FFB555F3A8600007FFB555F1230 VCRUNTIME140.dll!is_exception_typeof [<unknown file> @ <unknown line number>]
00007FFB555FEC0400007FFB555FE720 VCRUNTIME140.dll!_C_specific_handler [<unknown file> @ <unknown line number>]
00007FFB555F2C9400007FFB555F1230 VCRUNTIME140.dll!is_exception_typeof [<unknown file> @ <unknown line number>]
00007FFB555FF54100007FFB555FF4D0 VCRUNTIME140.dll!_CxxFrameHandler3 [<unknown file> @ <unknown line number>]
00007FFB6D5062BF00007FFB6D5061A0 ntdll.dll!_chkstk [<unknown file> @ <unknown line number>]
00007FFB6D3AE60300007FFB6D3AE380 ntdll.dll!RtlUnwindEx [<unknown file> @ <unknown line number>]
00007FFB555FF0CA00007FFB555FE720 VCRUNTIME140.dll!_C_specific_handler [<unknown file> @ <unknown line number>]
00007FFB555F18E500007FFB555F1230 VCRUNTIME140.dll!is_exception_typeof [<unknown file> @ <unknown line number>]
00007FFB555F1D0000007FFB555F1230 VCRUNTIME140.dll!is_exception_typeof [<unknown file> @ <unknown line number>]
00007FFB555F2D9000007FFB555F1230 VCRUNTIME140.dll!is_exception_typeof [<unknown file> @ <unknown line number>]
00007FFB555FF54100007FFB555FF4D0 VCRUNTIME140.dll!_CxxFrameHandler3 [<unknown file> @ <unknown line number>]
00007FFB6D50623F00007FFB6D5061A0 ntdll.dll!_chkstk [<unknown file> @ <unknown line number>]
00007FFB6D3B239700007FFB6D3B1E00 ntdll.dll!RtlLocateExtendedFeature [<unknown file> @ <unknown line number>]
00007FFB6D3AA95100007FFB6D3AA730 ntdll.dll!RtlRaiseException [<unknown file> @ <unknown line number>]
00007FFB6A1779DA00007FFB6A177950 KERNELBASE.dll!RaiseException [<unknown file> @ <unknown line number>]
00007FFB555F526700007FFB555F51D0 VCRUNTIME140.dll!CxxThrowException [<unknown file> @ <unknown line number>]
00007FFA8F93602400007FFA8F935EC0 c10_hip.dll!c10::cuda::c10_cuda_check_implementation [<unknown file> @ <unknown line number>]
00007FFA7A2B298D00007FFA7A2711F0 torch_python.dll!torch::lazy::GetPythonFrames [<unknown file> @ <unknown line number>]
00007FFA7A2B28A800007FFA7A2711F0 torch_python.dll!torch::lazy::GetPythonFrames [<unknown file> @ <unknown line number>]
00007FFA795FE3A800007FFA795F2090 torch_python.dll!THPPointer<THPStorage>::free [<unknown file> @ <unknown line number>]
00007FFB018372F800007FFB01836C24 python312.dll!PyUnicode_DecodeUTF7Stateful [<unknown file> @ <unknown line number>]
00007FFB01866A0200007FFB01866380 python312.dll!PyObject_Vectorcall [<unknown file> @ <unknown line number>]
00007FFB018663B500007FFB01866380 python312.dll!PyObject_Vectorcall [<unknown file> @ <unknown line number>]
00007FFB018678AD00007FFB01866FE0 python312.dll!PyEval_EvalFrameDefault [<unknown file> @ <unknown line number>]
00007FFB0187A74C00007FFB0187A020 python312.dll!PyObject_ClearManagedDict [<unknown file> @ <unknown line number>]
00007FFB01879F7D00007FFB01879ED0 python312.dll!PyEval_EvalCode [<unknown file> @ <unknown line number>]
00007FFB018387A800007FFB0183837C python312.dll!PyWeakref_NewProxy [<unknown file> @ <unknown line number>]
00007FFB0183864C00007FFB0183837C python312.dll!PyWeakref_NewProxy [<unknown file> @ <unknown line number>]
00007FFB0184984E00007FFB01849240 python312.dll!PySet_Add [<unknown file> @ <unknown line number>]
00007FFB018DEC7500007FFB018DEB50 python312.dll!PyObject_Call [<unknown file> @ <unknown line number>]
00007FFB018DEBBF00007FFB018DEB50 python312.dll!PyObject_Call [<unknown file> @ <unknown line number>]
00007FFB0186B20D00007FFB01866FE0 python312.dll!PyEval_EvalFrameDefault [<unknown file> @ <unknown line number>]
00007FFB01865A2C00007FFB018658B0 python312.dll!PyFunction_Vectorcall [<unknown file> @ <unknown line number>]
00007FFB0186644F00007FFB01866380 python312.dll!PyObject_Vectorcall [<unknown file> @ <unknown line number>]
00007FFB018DC54700007FFB018DC404 python312.dll!PyObject_CallMethodObjArgs [<unknown file> @ <unknown line number>]
00007FFB018DC47400007FFB018DC404 python312.dll!PyObject_CallMethodObjArgs [<unknown file> @ <unknown line number>]
00007FFB01836BF900007FFB01836A94 python312.dll!PySys_GetObject [<unknown file> @ <unknown line number>]
00007FFB01840CE600007FFB018409C0 python312.dll!PyImport_ImportModuleLevelObject [<unknown file> @ <unknown line number>]
00007FFB018DA99800007FFB018DA8C8 python312.dll!PyErr_Clear [<unknown file> @ <unknown line number>]
00007FFB0186CB4000007FFB01866FE0 python312.dll!PyEval_EvalFrameDefault [<unknown file> @ <unknown line number>]
00007FFB0187A74C00007FFB0187A020 python312.dll!PyObject_ClearManagedDict [<unknown file> @ <unknown line number>]
00007FFB01879F7D00007FFB01879ED0 python312.dll!PyEval_EvalCode [<unknown file> @ <unknown line number>]
00007FFB018387A800007FFB0183837C python312.dll!PyWeakref_NewProxy [<unknown file> @ <unknown line number>]
00007FFB0183864C00007FFB0183837C python312.dll!PyWeakref_NewProxy [<unknown file> @ <unknown line number>]
00007FFB0184984E00007FFB01849240 python312.dll!PySet_Add [<unknown file> @ <unknown line number>]
00007FFB018DEC7500007FFB018DEB50 python312.dll!PyObject_Call [<unknown file> @ <unknown line number>]
00007FFB018DEBBF00007FFB018DEB50 python312.dll!PyObject_Call [<unknown file> @ <unknown line number>]
00007FFB0186B20D00007FFB01866FE0 python312.dll!PyEval_EvalFrameDefault [<unknown file> @ <unknown line number>]
00007FFB01865A2C00007FFB018658B0 python312.dll!PyFunction_Vectorcall [<unknown file> @ <unknown line number>]

Fatal Python error: Aborted

Stack (most recent call first):
  File "C:\comfyui-rocm\python_env\Lib\site-packages\torch\cuda\memory.py", line 847 in mem_get_info
  File "C:\comfyui-rocm\comfy\model_management.py", line 242 in get_total_memory
  File "C:\comfyui-rocm\comfy\model_management.py", line 257 in <module>
  File "<frozen importlib._bootstrap>", line 488 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 999 in exec_module
  File "<frozen importlib._bootstrap>", line 935 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1331 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1360 in _find_and_load
  File "C:\comfyui-rocm\execution.py", line 17 in <module>
  File "<frozen importlib._bootstrap>", line 488 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 999 in exec_module
  File "<frozen importlib._bootstrap>", line 935 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1331 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1360 in _find_and_load
  File "C:\comfyui-rocm\main.py", line 198 in <module>

Extension modules: sqlalchemy.cyextension.collections, sqlalchemy.cyextension.immutabledict, sqlalchemy.cyextension.processors, sqlalchemy.cyextension.resultproxy, sqlalchemy.cyextension.util, greenlet._greenlet, markupsafe._speedups, yaml._yaml, PIL._imaging, multidict._multidict, yarl._quoting_c, propcache._helpers_c, aiohttp._http_writer, aiohttp._http_parser, aiohttp._websocket.mask, aiohttp._websocket.reader_c, frozenlist._frozenlist, charset_normalizer.md, charset_normalizer.cd, requests.packages.charset_normalizer.md, requests.packages.chardet.md, requests.packages.charset_normalizer.cd, requests.packages.chardet.cd, numpy._core._multiarray_umath, numpy.linalg._umath_linalg, torch._C, torch._C._dynamo.autograd_compiler, torch._C._dynamo.eval_frame, torch._C._dynamo.guards, torch._C._dynamo.utils, torch._C._fft, torch._C._linalg, torch._C._nested, torch._C._nn, torch._C._sparse, torch._C._special, numpy.random._common, numpy.random.bit_generator, numpy.random._bounded_integers, numpy.random._pcg64, numpy.random._generator, numpy.random._mt19937, numpy.random._philox, numpy.random._sfc64, numpy.random.mtrand, psutil._psutil_windows (total: 46)
Для продолжения нажмите любую клавишу . . .

**Comment by patientx** [comment]
can you do a clean install with --debug (install.bat --debug) and attach the log here?

**Comment by 0xDELUXA** [comment]
Try to activate the virtual environment, then run:
```powershell
python -c "
import torch
print('= PyTorch Info =')
print(f'PyTorch version: {torch.__version__}')
print(f'HIP version:     {torch.version.hip}')

print()
print('= GPU =')
print(f'Available:    {torch.cuda.is_available()}')
print(f'Device count: {torch.cuda.device_count()}')

if torch.cuda.is_available():
    for i in range(torch.cuda.device_count()):
        props = torch.cuda.get_device_properties(i)
        print(f'Device {i}:          {torch.cuda.get_device_name(i)}')
        print(f'  VRAM:              {props.total_memory / 1024**3:.2f} GB')
        print(f'  Multi-processors:  {props.multi_processor_count}')
        print(f'  Compute cap:       {props.major}.{props.minor}')
    print()
    print('= Memory =')
    free, total = torch.cuda.mem_get_info()
    print(f'Free:  {free / 1024**3:.2f} GB')
    print(f'Total: {total / 1024**3:.2f} GB')
    print()
    print('= MIOpen =')
    print(f'Available: {torch.backends.cudnn.is_available()}')
    print(f'Version:   {torch.backends.cudnn.version()}')
else:
    print('No GPU detected')
"
```
